packages/ui/react-ui-table/src/components/Table/relations.stories.tsx

```diff
@@ -47,7 +48,7 @@ const useTestModel = <T extends BaseObject & HasId>(schema: BaseSchema<T>, count
     }
 
     // TODO(burdon): Just pass in view? Reuse same jsonSchema instance? View determines if mutable, etc.
-    return new ViewProjection(schema.jsonSchema, table.view.target);
+    return new ViewProjection(toJsonSchema(schema), table.view.target);
   }, [schema, table]);
```

Comment:
Since we're calling `toJsonSchema` in both places we should probably memoize that then add it as a dependency to these two memos.